examples/storage_volume_templates.py

```diff
@@ -41,12 +41,11 @@
 
 options = {
     "name": "OneView Test Volume Template",
-    "provisioning":
-        {
-            "capacity": "235834383322",
-            "provisionType": "Thin",
-            "storagePoolUri": ""
-        }
+    "provisioning": {
```

Comment:
i checked the spec, and there its provisioningType not provisioning. Is it a doc defect?

Reply:
Yes, Its expecting provisioning property and its not documented.